.travis.yml

```diff
@@ -2,13 +2,17 @@ language: go
 
 go:
   - 1.3
+  - 1.4
 
 install:
-  - go get code.google.com/p/go.tools/cmd/cover
+  - go get golang.org/x/tools/cmd/cover
+  - go get -u github.com/golang/lint/golint
+  - ./hack/verify-gofmt.sh
+  - ./hack/verify-golint.sh || true
   - ./hack/build-go.sh
 
 script:
-  - ./hack/test-go.sh
+  - KUBE_RACE="-race" KUBE_COVER="-cover -covermode=atomic" ./hack/test-go.sh "" -p=4
```

Comment:
Actually we're not, I've renamed all of them to be `STI_*`, PR with fix to above https://github.com/openshift/source-to-image/pull/141